src/redux/configureStore.js

```diff
@@ -0,0 +1,16 @@
+import { combineReducers } from 'redux';
+import { configureStore } from '@reduxjs/toolkit';
```

Comment:
- [ ] You're currently importing `configureStore` from `@reduxjs/toolkit;` however, in this milestone, you are expected to 
import `createStore` from `redux`. Please make this change 👍

Reply:
`CreateStore` is deprecated. attached is a screenshot of eslint validation.

<img width="1436" alt="Screenshot 2022-05-03 at 02 06 30" src="https://user-images.githubusercontent.com/31113416/166340560-d3211841-a728-4ef8-850d-6d63c0ef0c33.png">

Reply:
You are right. Please go ahead and request another review 👍